components/SearchBox/components/SearchResult.tsx

```diff
@@ -0,0 +1,37 @@
+import type { Result } from '@orama/orama';
+import type { FC } from 'react';
+
+import type { SearchDoc } from '@/components/SearchBox/components/SearchBox';
+import { highlighter } from '@/components/SearchBox/lib/orama';
+import { pathToBreadcrumbs } from '@/components/SearchBox/lib/utils';
+
+import styles from './index.module.css';
+
+type SearchResultProps = {
+  hit: Result<SearchDoc>;
+  searchTerm: string;
+};
+
+export const SearchResult: FC<SearchResultProps> = props => {
+  return (
+    <a
+      key={props.hit.id}
+      href={`/en/${props.hit.document.path}`}
+      className={styles.fulltextSearchResult}
+    >
+      <div
+        className={styles.fulltextSearchResultTitle}
+        dangerouslySetInnerHTML={{
```

Comment:
We drastically discourage using this.

Reply:
I understand, but since the highlighter generates HTML as output, there's no way around that I am aware of. Also, the source documents are all within this repository (this is not user-submitted content), and therefore should be safe